pre-docs/bolt.ditamap

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
+<map id="boltmap" title="Bolt">
+    <topicref href="bolt.md" format="markdown"/>
+    <topicref href="bolt_release_notes.md" format="markdown"/>
```

Comment:
It looks like this page was deleted? Does this build?

Reply:
I've re-added the page and reworked it a little to reflect our current organization, including the use of `CHANGELOG.md`.